--- setup.py	2019-05-14 13:20:29.622723344 -0500
+++ setup-fixed.py	2019-05-14 12:25:52.303777000 -0500
@@ -77,8 +77,8 @@
 
 # Get geos_version from GEOS dynamic library, which depends on
 # GEOS_LIBRARY_PATH and/or GEOS_CONFIG environment variables
-from shapely._buildcfg import geos_version_string, geos_version, \
-        geos_config, get_geos_config
+# from shapely._buildcfg import geos_version_string, geos_version, \
+#         geos_config, get_geos_config
 
 logging.basicConfig()
 log = logging.getLogger(__file__)
