diff -crB a/qpid-java-0.30/broker/bin/qpid-server b/qpid-java-0.30/broker/bin/qpid-server
*** a/qpid-java-0.30/broker/bin/qpid-server	2014-07-22 05:09:32.000000000 -0500
--- b/qpid-java-0.30/broker/bin/qpid-server	2015-06-24 10:24:42.268576948 -0500
***************
*** 23,30 ****
  fi
  
  if [ -z "$QPID_WORK" ]; then
!     echo "Setting QPID_WORK to $HOME as default"
!     QPID_WORK=$HOME
  fi
  
  # Set to help us get round the manifold problems of ps/pgrep on various
--- 23,29 ----
  fi
  
  if [ -z "$QPID_WORK" ]; then
!     QPID_WORK=$HOME_HOME
  fi
  
  # Set to help us get round the manifold problems of ps/pgrep on various
***************
*** 37,43 ****
  QPID_LIBS="$QPID_HOME/lib/*:$QPID_HOME/lib/plugins/*:$QPID_HOME/lib/opt/*"
  
  # Set other variables used by the qpid-run script before calling
! export JAVA=java \
         JAVA_VM=-server \
         JAVA_MEM=-Xmx2g \
         JAVA_GC="-XX:+UseConcMarkSweepGC -XX:+HeapDumpOnOutOfMemoryError" \
--- 36,42 ----
  QPID_LIBS="$QPID_HOME/lib/*:$QPID_HOME/lib/plugins/*:$QPID_HOME/lib/opt/*"
  
  # Set other variables used by the qpid-run script before calling
! export JAVA=/awips2/java/bin/java \
         JAVA_VM=-server \
         JAVA_MEM=-Xmx2g \
         JAVA_GC="-XX:+UseConcMarkSweepGC -XX:+HeapDumpOnOutOfMemoryError" \
diff -crB a/qpid-java-0.30/broker-core/src/main/java/org/apache/qpid/server/binding/BindingImpl.java b/qpid-java-0.30/broker-core/src/main/java/org/apache/qpid/server/binding/BindingImpl.java
*** a/qpid-java-0.30/broker-core/src/main/java/org/apache/qpid/server/binding/BindingImpl.java	2014-08-25 10:35:10.000000000 -0500
--- b/qpid-java-0.30/broker-core/src/main/java/org/apache/qpid/server/binding/BindingImpl.java	2015-06-24 10:18:56.496221120 -0500
***************
*** 227,232 ****
--- 227,233 ----
              }
              getEventLogger().message(_logSubject, BindingMessages.DELETED());
          }
+         deleted();
          _state = State.DELETED;
      }
  
diff -crB a/qpid-java-0.30/common/bin/qpid-run b/qpid-java-0.30/common/bin/qpid-run
*** a/qpid-java-0.30/common/bin/qpid-run	2013-11-13 18:02:20.000000000 -0600
--- b/qpid-java-0.30/common/bin/qpid-run	2015-06-24 10:23:14.444503462 -0500
***************
*** 62,69 ****
  fi
  
  if [ -z "$QPID_WORK" ]; then
!     log $INFO Setting QPID_WORK to $HOME as default
!     QPID_WORK=$HOME
  fi
  
  if [ -z "$JAVA" ]; then
--- 62,68 ----
  fi
  
  if [ -z "$QPID_WORK" ]; then
!     QPID_WORK=$HOME_HOME
  fi
  
  if [ -z "$JAVA" ]; then
