*** a/postgis-2.0.6/postgis/lwgeom_accum.c	2012-02-27 16:08:12.000000000 -0600
--- b/postgis-2.0.6/postgis/lwgeom_accum.c	2016-07-21 15:20:53.184106704 -0500
***************
*** 105,124 ****
  		        (errcode(ERRCODE_INVALID_PARAMETER_VALUE),
  		         errmsg("could not determine input data type")));
  
  	if (fcinfo->context && IsA(fcinfo->context, AggState))
  		aggcontext = ((AggState *) fcinfo->context)->aggcontext;
! #if POSTGIS_PGSQL_VERSION == 84
! 
  	else if (fcinfo->context && IsA(fcinfo->context, WindowAggState))
  		aggcontext = ((WindowAggState *) fcinfo->context)->wincontext;
! #endif
! #if POSTGIS_PGSQL_VERSION > 84
! 
! 	else if (fcinfo->context && IsA(fcinfo->context, WindowAggState))
! 		aggcontext = ((WindowAggState *) fcinfo->context)->aggcontext;
! #endif
! 
  	else
  	{
  		/* cannot be called directly because of dummy-type argument */
  		elog(ERROR, "array_agg_transfn called in non-aggregate context");
--- 105,123 ----
  		        (errcode(ERRCODE_INVALID_PARAMETER_VALUE),
  		         errmsg("could not determine input data type")));
  
+ #if POSTGIS_PGSQL_VERSION == 84
  	if (fcinfo->context && IsA(fcinfo->context, AggState))
+ 	{
  		aggcontext = ((AggState *) fcinfo->context)->aggcontext;
! 	}
  	else if (fcinfo->context && IsA(fcinfo->context, WindowAggState))
+ 	{
  		aggcontext = ((WindowAggState *) fcinfo->context)->wincontext;
! 	}
  	else
+ #else
+ 	if ( ! AggCheckCallContext(fcinfo, &aggcontext) )
+ #endif
  	{
  		/* cannot be called directly because of dummy-type argument */
  		elog(ERROR, "array_agg_transfn called in non-aggregate context");
